app/models/follower_followed.rb

```diff
@@ -1,6 +1,4 @@
 class FollowerFollowed < ApplicationRecord
   validates :follower_id,
             :followed_id, presence: true
-
-  belongs_to :user
```

Comment:
This was breaking tests, so I'll be doing some research as to why that might be, though I believe it does belong there.